README.md

```diff
@@ -1,7 +1,8 @@
 ## Generate cbindgen for go
 
 ````
-cbindgen --config cbindgen.toml  --output api/bindings.h
+cd libgo_owasm
+cbindgen --config cbindgen.toml  --output ../api/bindings.h
```

Comment:
Did you use this command to gen binding.h or generate by rust?

Reply:
I generated it by rust. It will automatically generate when you build the rust project.
However, you are able to use this command to generate as well.

Reply:
already updated readme to use rust build command.